OpenDreamRuntime/DreamRuntime.cs

```diff
@@ -191,6 +191,7 @@ public class DreamRuntime
             connection.SendPacket(new PacketConnectionResult(true, null, interfaceResource.ReadAsString()));
             connection.SendPacket(new PacketFullGameState(StateManager.FullState, connection.CKey));
 
+            client.SetVariable("connection", new DreamValue("seeker"));
```

Comment:
This leaves it possible for DM code to set `client.connection`. This should be implemented as a variable getter in `DreamMetaObjectClient`